### DISCUSSION TITLE
Complejidad de funciones que usan la librería estándar

### DISCUSSION THREAD

**Discussion by IoachimusRoderici** [INGCOM-UNRN-P1/cursada-2023#213] [Preguntas y respuestas] [answered]
En varios de los ejercicios del tp 10 se usan funciones de la librería como `fprintf`, `fscanf`, o `malloc`.

¿Cómo calculo la complejidad en estos casos?¿Necesitaría conocer la complejidad de las funciones llamadas, no?

**Comment by martinvilu** [comment, marked as answer]
Toma las funciones de la librería como `1`.
